geth/utils/validation.py

```diff
@@ -135,13 +145,37 @@ def validate_genesis_data(genesis_data: GenesisDataTypedDict) -> bool:
             raise PyGethValueError(
                 f"error while validating genesis_data config field: {e}"
             )
+
+    return True
```

Comment:
nit: I think we can probably leave this out and return `None` in the signature. Generally I don't expect validation methods to return anything, except raise if appropriate.

Reply:
fixed! makes sense